docs/modules/operation/pages/deep-dive/admin/configuration/data-collection.adoc

```diff
@@ -1,7 +1,7 @@
 
 = Data Collection
 
-When enabled, the `Data Choices` module collects anonymous use statistics.
+When enabled, the Data Choices module collects anonymous use statistics.
 It publishes them on system startup and every 24 hours after.
```

Comment:
Could possibly add reference to https://stats.opennms.com/ that shows the aggregated stats that have been collected.